.gitattributes

```diff
@@ -0,0 +1,19 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto eol=lf
```

Comment:
I suggest leaving text=auto and not eol=lf

People working under Windows might get into trouble with this.

Reply:
Done. Wasn't sure which is better.